src/main/java/fr/xephi/authme/settings/properties/RestrictionSettings.java

```diff
@@ -193,6 +193,22 @@ public final class RestrictionSettings implements SettingsHolder {
     public static final Property<Set<String>> UNRESTRICTED_INVENTORIES =
         newLowercaseStringSetProperty("settings.unrestrictions.UnrestrictedInventories");
 
+    @Comment({
+        "While in unregistered/logged out state, should players be set to spectator mode and",
+        "forced to spectate within a spawn point invisible armor stand, for 0 movement and head",
+        "pitch + yaw? may be more effective than 'allowMovement' at locking the player in place."
+    })
+    public static final Property<Boolean> SPECTATE_STAND_LOGIN =
+        newProperty("settings.restrictions.spectateStandLogin.enabled", false);
+
+    @Comment("Head Yaw position for 'spectateStandLogin'.")
+    public static final Property<Double> HEAD_YAW =
+        newProperty("settings.restrictions.spectateStandLogin.headYaw", 0.0f);
+
+    @Comment("Head Pitch position for 'spectateStandLogin'.")
+    public static final Property<Double> HEAD_PITCH =
```

Comment:
I think this should work with the current version of ConfigMe (ignore this comment if it doesn't)

```suggestion
    public static final Property<Float> HEAD_PITCH =
```

Also the naming is a bit weird, I don't know that `RestrictionSettings#HEAD_PITCH` refers to the spectate stand feature, and as a user I'm not sure I'll understand what this setting represents. Is it possible to elaborate in the comment? I would make a suggestion but I'm actually not really sure what the yaw & pitch settings are for

Reply:
I don't know If i understood you correctly, but `HEAD_PITCH` and `HEAD_YAW` are under `spectateStandLogin` so It's obvious what are these settings for.

Reply:
Thanks for changing the comment. I also meant the field name, `RestrictionSettings.HEAD_PITCH` in the code sounds very generic. Could we maybe name it `SPECTATE_HEAD_PITCH`, or maybe you have a better idea?

Reply:
Renamed.